HEC.FDA.Model/metrics/AlternativeResults.cs

```diff
@@ -108,22 +106,22 @@ public List<string> GetDamageCategories()
         /// <summary>
         /// This method returns the mean of the average annual equivalent damage for the given damage category, asset category, impact area combination 
         /// The level of aggregation of the mean is determined by the arguments used in the method
-        /// For example, if you wanted mean AAEQ damage for residential, impact area 2, all asset categories, then the method call would be as follows:
-        /// double meanEAD = MeanAAEQDamage(damageCategory: "residential", impactAreaID: 2);
+        /// For example, if you wanted mean eqad damage for residential, impact area 2, all asset categories, then the method call would be as follows:
+        /// double meanEAD = SampleMeanEqad(damageCategory: "residential", impactAreaID: 2);
         /// </summary>
         /// <param name="damageCategory"></param> either residential, commercial, etc...the default is null
         /// <param name="assetCategory"></param> either structure, content, etc...the default is null
         /// <param name="impactAreaID"></param> the default is the null value utilities.IntegerConstants.DEFAULT_MISSING_VALUE
-        /// <returns></returns>The mean of aaeq damage
-        public double SampleMeanAAEQDamage(int impactAreaID = utilities.IntegerGlobalConstants.DEFAULT_MISSING_VALUE, string damageCategory = null, string assetCategory = null)
+        /// <returns></returns>The mean of EqAD 
+        public double SampleMeanEqad(int impactAreaID = utilities.IntegerGlobalConstants.DEFAULT_MISSING_VALUE, string damageCategory = null, string assetCategory = null)
```

Comment:
this is semantics....

I wonder if there is better naming convention here. When I read "SampleMeanEqad" I read "sample" as a verb even though you mean "sample" as a modifier. 

If we were in Java land, this might be straightforward to say GetSampleMeanEqad

anyway, semantics, and there are xml comments